packages/aws-cdk-lib/aws-codepipeline-actions/lib/ecs/deploy-action.ts

```diff
@@ -80,14 +80,15 @@ export class EcsDeployAction extends Action {
   protected bound(_scope: Construct, _stage: codepipeline.IStage, options: codepipeline.ActionBindOptions):
   codepipeline.ActionConfig {
     // permissions based on CodePipeline documentation:
-    // https://docs.aws.amazon.com/codepipeline/latest/userguide/how-to-custom-role.html#how-to-update-role-new-services
```

Comment:
Is the link change just to point to the more specific section?

Reply:
The previous link is invalid and redirect to the welcome page.